src/SprykerEco/Zed/PunchoutCatalogs/Persistence/PunchoutCatalogsEntityManager.php

```diff
@@ -62,6 +62,8 @@ public function updatePunchoutCatalogConnection(PunchoutCatalogConnectionTransfe
                 $punchoutCatalogConnectionEntity
             );
 
-        return (bool)$punchoutCatalogConnectionEntity->save();
+        $punchoutCatalogConnectionEntity->save();
```

Comment:
if you don't expect that something will be updated in DB than you don't need return type here and if must be in the business layer.